docs: add more details for doc and change examples to must have a view (#238)
cmudig/draco2: docs/facts/facet.md

# Facets

With the facet operator, we can partition a dataset by a field and create a view for each field. The resulting chart is often called a small multiples chart.

## Facet Properties

`(facet,channel)`
: The facet channel. Can be one of `col` and `row`.

`(facet,field)`
: The facet field.

`(facet,binning)`

> : Binning a numeric field for faceting. Numeric fields have to be binned for faceting.

Reviewer: ```suggestion
: Binning a numeric field for faceting. A facet field has to be binned if it's numeric.```
